packages/instantsearch.js/src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -437,7 +437,7 @@ describe('insights', () => {
       );
     });
 
-    it('warns when userToken is not set', () => {
```

Comment:
Not sure if this is needed now, since an anonymous token would always be set for queries.

Reply:
yes, there's no realistic way to get this warning anymore. Still fine to keep at as a worst-case warning but I don't see how you could _not_ have a token anymore with these changes.

Reply:
the test should be removed, not skipped :)